cdi-platform-extra-tck/pom.xml

```diff
@@ -0,0 +1,453 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>fish.payara.jakarta.tests.tck</groupId>
+        <artifactId>tck</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>cdi-platform-extra-tck</artifactId>
+
+    <properties>
+        <cdi.tck-4-1.version>4.1.0</cdi.tck-4-1.version>
+        <!-- The CDI EE Platform Integration TCK Version -->
+        <tck.version>11.0.0</tck.version>
+        
+        <maven.build.cache.enabled>false</maven.build.cache.enabled>
+
+        <!-- Weld uses 2 versions numbers. Some artefact use one version, other the second. -->
+        <weld.version>6.0.1.Final</weld.version>
+        <weld.version2>6.0.Final</weld.version2>
```

Comment:
We should pull these from the payara-bom
```suggestion
```

Reply:
```
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project fish.payara.jakarta.tests.tck:cdi-platform-extra-tck:1.0-SNAPSHOT (/home/luise/git/tck/jakartaee-10-tck-runners/cdi-platform-extra-tck/pom.xml) has 3 errors
[ERROR]     'dependencies.dependency.version' for org.jboss.weld:weld-spi:jar is missing. @ line 102, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.weld:weld-core-impl:jar is missing. @ line 113, column 21
[ERROR]     'dependencies.dependency.version' for org.jboss.weld.module:weld-web:jar is missing. @ line 124, column 21
```